scripts/main/multiselect.js

```diff
@@ -256,7 +256,7 @@ multiselect.show = function(e) {
 		multiselect.clearSelection()
 	}
 
-	multiselect.position.top    = e.pageY;
+	multiselect.position.top    = e.pageY+1;
```

Comment:
This is a bit of a hack, but seems to fix the back button. Better solutions welcome!